packages/EasyCore/src/Bridge/Symfony/ApiPlatform/DataPersister/AbstractDataPersister.php

```diff
@@ -0,0 +1,69 @@
+<?php
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Bridge\Symfony\ApiPlatform\DataPersister;
+
+use ApiPlatform\Core\DataPersister\DataPersisterInterface;
+
+abstract class AbstractDataPersister implements DataPersisterInterface
+{
+    /**
+     * @var mixed
+     */
+    protected $dataPersister;
+
+    /**
+     * AbstractDataPersister constructor.
+     *
+     * @param mixed $dataPersister
+     */
+    public function __construct($dataPersister)
```

Comment:
Same here the argument should be a `DataPersiterInterface` and called `$decorated`.